README.md

```diff
@@ -217,70 +223,73 @@ Conversion using a live exchange rate with 500 seconds nowishSecs,
 this conversion will be done via HTTP request:
 
 ```scala
-
 // 9.99 GBP => EUR
-val fx = Forex(ForexConfig(nowishSecs = 500, nowishCacheSize = 0), OerClientConfig(appId, DeveloperAccount))
-val priceInEuros = fx.convert(9.99, "GBP").to("EUR").nowish
+val priceInEuros = for {
+  fx     <- Forex.getForex[IO](ForexConfig("YOUR_API_KEY", DeveloperAccount, nowishSecs = 500, nowishCacheSize = 0))
+  result <- fx.convert(9.99, CurrencyUnit.GBP).to(CurrencyUnit.EUR).nowish
+} yield result
 ```
 
 #### 3.3.4 Latest-prior EOD rate
 
-Conversion using the latest EOD (end-of-date) rate prior to your event _(cacheing available)_:
+Conversion using the latest EOD (end-of-date) rate prior to your event _(caching available)_:
 
 ```scala
-import org.joda.time.{DateTime, DateTimeZone}
-
 // 10000 GBP => JPY at the end of 12/03/2011
-val fx = Forex(ForexConfig(), OerClientConfig(appId, DeveloperAccount))
-val tradeDate = new DateTime(2011, 3, 13, 11, 39, 27, 567, DateTimeZone.forID("America/New_York"))
-val tradeInYen = fx.convert(10000, "GBP").to("JPY").at(tradeDate)                   
+val tradeDate = ZonedDateTime.of(2011, 3, 13, 11, 39, 27, 567, ZoneId.of("America/New_York"))
+val tradeInYen = for {
+  fx     <- forex
+  result <- fx.convert(10000, CurrencyUnit.GBP).to(CurrencyUnit.JPY).at(tradeDate)
+} yield result
 ```
 
 #### 3.3.5 Latest-post EOD rate
 
-Lookup the latest EOD (end-of-date) rate following your event _(cacheing available)_:
+Lookup the latest EOD (end-of-date) rate following your event _(caching available)_:
 
 ```scala
-import org.joda.time.{DateTime, DateTimeZone}
-import com.snowplowanalytics.forex.EodRoundUp
-
 // 10000 GBP => JPY at the end of 13/03/2011
-val fx = Forex(ForexConfig(getNearestDay = EodRoundUp), OerClientConfig(appId, DeveloperAccount))
-val tradeDate = new DateTime(2011, 3, 13, 11, 39, 27, 567, DateTimeZone.forID("America/New_York"))
-val usd2yen = fx.convert(10000, "GBP").to("JPY").at(tradeDate)
+val tradeDate = ZonedDateTime.of(2011, 3, 13, 11, 39, 27, 567, ZoneId.of("America/New_York"))
+val usd2jpy = for {
+  fx     <- Forex.getForex[IO](ForexConvig("YOU_API_KEY", DeveloperAccount, getNearestDay = EodRoundUp))
```

Comment:
* `YOUR_API_KEY` -> `YOUR_API_KEY`
* `ForexConvig` -> `ForexConfig`
* Also, I think `YOUR_API_KEY` is misleading as it should be something like "app id"